File: CodeMeter/CodeMeter.HttpService/Models/Configuration.cs
using System.ComponentModel.DataAnnotations;

namespace CodeMeter.HttpService.Models
{
    public class Configuration : Entity
    {
        public int CheckInterval { get; set; }
        public int NotificationInterval { get; set; }
        public bool CheckRunning { get; set; }
        public bool SendSms { get; set; }
        public bool SendEmail { get; set; }
        [StringLength(255)]
        public string Recepient { get; set; }
        [StringLength(255)]
        public string Sender { get; set; }

        [StringLength(255)]
        public string Smtp { get; set; }
        public int Port { get; set; }
        [StringLength(255)]
        public string Username { get; set; }
        [StringLength(255)]
        public string Password { get; set; }
        public bool Ssl { get; set; }
    }
}

File: CodeMeter/CodeMeter.Client/app/viewmodels/configuration.js
define(["jquery", "knockout", "durandal/app", "durandal/system", "plugins/router", "services/data", "komapping"], function ($, ko, app, system, router, data, mapper) {
    var
    // Properties
        cfg = mapper.fromJS({
        ID: '',
        CheckInterval: 0,
        CheckRunning: false,
        NotificationInterval: 15,
        Password: '',
        Port: 26,
        Recepient: '',
        SendEmail: false,
        SendSms: false,
        Sender: '',
        Smtp: '',
        Ssl: false,
        Username: ''
    }),
    isBusy = ko.observable(false),
    // Handlers
    save = function() {
        isBusy(true);
        data.updateConfiguration(cfg).done(function(){
            
        }).fail(function(){
            
        }).always(function(){
            isBusy(false);
        })
    },
    // Lifecycle

    activate = function () {
        return data.getConfiguration().done(function (configuration) {
            mapper.fromJS(configuration, cfg);
        })
    },

    deactivate = function () {};

    return {
        // Place your public properties here
        isBusy: isBusy,
        cfg: cfg,
        save: save,
        activate: activate,
        deactivate: deactivate
    };
});